its/ruling/src/test/resources/expected/python-S2638.json

```diff
@@ -39,9 +39,6 @@
 'project:buildbot-0.8.6p1/buildbot/pbutil.py':[
 75,
 ],
-'project:buildbot-0.8.6p1/buildbot/util/__init__.py':[
-166,
```

Comment:
This is now an FN we do not detect anymore that tzname needs a dt object as parameter